Update RESTful API (#1997)
pytorch/fairseq: examples/simultaneous_translation/docs/evaluation.md

```
For text, the segment is a detokenized word, while for speech, it is a list of numbers.

## Client

> The client will load model, get source tokens from server and send translated target tokens to server. 

Reviewer: ```suggestion
The client will load the model, get source tokens or speech segments from the server and send translated tokens to server. 
```

> We have an out-of-box implementation of client which could save time from configuring the communication between the server and client.

Reviewer: ```suggestion
We have an out-of-box implementation of the client which could save time from configuring the communication between the server and the client.
```

> It's in fairseq repository but it can be applied to an arbitary toolkit.

Reviewer: ```suggestion
The implementation is in the fairseq repository but it can be applied to an arbitary toolkit.
```

 
```

> The variable **states** is the context for the model to make the decision.

Reviewer: ```suggestion
The variable **states** is the context for the model to make a decision.
```

> You could customize your one states format

Reviewer: ```suggestion
You could customize your states
```

 `init_states` will be called at beginning of translating every sentnece. 

> `update_states` will be called everytime a new segement was requested from server

Reviewer: ```suggestion
`update_states` will be called every time a new segment or token was requested from server
```

The `policy` function returns the action you make given the current states. 

> The actions can be one of the tree as follow.

Reviewer: ```suggestion
The actions can be one of the following:
```

|Action|Content|
|:---:|:---:|
|Request new word (Text)| ```{key: "GET", value: None}```|
|Request new utterence (Speech) | ```{key: "GET", value: {"segment_size": segment_size}}```|
|Predict word "W"| ```{key: "SEND", value: "W"}```|

Here is an example of how to develop a customized agent. 

> First of all, a name needs to be registered, here is "my_agent". 

Reviewer: ```suggestion
First of all, a name needs to be registered, for example "my_agent". 
```
